sql/core/src/test/scala/org/apache/spark/sql/DataFrameComplexTypeSuite.scala

```diff
@@ -26,6 +26,20 @@ import org.apache.spark.sql.test.SharedSQLContext
 class DataFrameComplexTypeSuite extends QueryTest with SharedSQLContext {
   import testImplicits._
 
+  test("primitive type on array") {
+    val df = sparkContext.parallelize(Seq(1, 2), 1).toDF("v")
```

Comment:
Yes, this is what I want to do. As other test cases did, I use ```sparkContext.parallelize()``` that can explicitly specify # of partitions.